Source/WebInspectorUI/UserInterface/Views/TreeOutline.js

```diff
@@ -449,11 +428,12 @@ WI.TreeOutline = class TreeOutline extends WI.Object
         if (representedObject.__treeElementIdentifier) {
             // If this representedObject has a tree element identifier, and it is a known TreeElement
             // in our tree we can just return that tree element.
-            var elements = this._knownTreeElements[representedObject.__treeElementIdentifier];
+            let elements = this._knownTreeElements.get(representedObject.__treeElementIdentifier);
             if (elements) {
```

Comment:
I feel like we could remove `if (elements)` by undefined-chaining instead: `for let (element of elements ?? [])`

Reply:
i don't think we've done that much in the past since it adds a _lot_ of extra code behind the scenes, and given how performance sensitive this code could be im inclined to keep it as-is for now